aspen/__main__.py

```diff
@@ -23,5 +23,29 @@
 
 from . import serve, website
 
+import logging.config
+
+logging_cfg = {
+    'version': 1,
+    'formatters': {
+        'threadinfo': {
+            'format': "%(asctime)s pid-%(process)d thread-%(thread)d (%(threadName)s) %(levelname)s: %(message)s"
+        }
+    },
+    'handlers': {
+        'console': {
+            'class': 'logging.StreamHandler',
+            'formatter': 'threadinfo',
+            'level': 'INFO',
+            'stream': 'ext://sys.stderr'
+        }
+    },
+    'root': {
+        'handlers': [ 'console' ]
+    }
+}
+
+logging.config.dictConfig(logging_cfg)
```

Comment:
This is in `__main__` so it's only called via `python -m aspen` - ie. the dev server.